stories/form.fmf

```diff
@@ -0,0 +1,16 @@
+/machine-readable:
+    story:
+        As a Testing Farm I want to get a machine readable output from the service.
+
+    description: |
+        Testing Farm needs to be able to get a machine readable output from the service, various formats could be
+        supported, such as JSON, YAML, XML, etc.
```

Comment:
Let's clearly state that `json` and `yaml` will be supported. I don't think we have to provide more flexibility.